src/Files.App/Helpers/Win32/Win32Helper.WindowManagement.cs

```diff
@@ -4,8 +4,10 @@
 using Microsoft.UI.Input;
 using Microsoft.UI.Xaml;
 using System.Reflection;
+using System.Runtime.InteropServices;
 using Windows.Win32;
 using Windows.Win32.UI.WindowsAndMessaging;
+using static Vanara.PInvoke.User32;
```

Comment:
We're trying to avoid new references to Vanara. While it's not absolutely required, it would be preferable if this can be achieved differently. Refer to issue #15000 for more details.

Reply:
Get it. Finished in 1aa2cd6.